ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -58,6 +59,7 @@ type JobSubmissionMode string
 const (
 	K8sJobMode JobSubmissionMode = "K8sJobMode" // Submit job via Kubernetes Job
 	HTTPMode   JobSubmissionMode = "HTTPMode"   // Submit job via HTTP request
+	NoneMode   JobSubmissionMode = "NoneMode"   // Don't submit job
```

Comment:
Brainstorming some alternative names for this mode:
* UserMode
* LocalMode

Reply:
I think both are better than NoneMode. I personally prefer UserMode. @kevin85421 What do you think?

Reply:
UserMode sounds good to me

Reply:
Renamed to UserMode